web.html

```diff
@@ -1,125 +1,224 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="main.css">
+    <link rel="stylesheet" href="./cssDesktop/laptop.css" media="(min-width: 1200px)">
     <title>YPF ruta</title>
 </head>
+
 <body>
     <header>
-        <nav>
-            <ul>
-                <li>Nabvar</li>
-            </ul>
-        </nav>
-        <h1>CONECTATE CON LA RUTA DE LAS SOLUCIONES</h1>
-        <p>Ahora es más fácil gestionar tu flota.</p>
-        <p>Tranformamos YPF Ruta en una solución integral para la movilidad y gestión de tus vehículos</p>
-        <a href="">Quiero adherirme</a>
-        <img src="" alt="">
-        <img src="" alt="">
-        <ul>
-            <li><a href=""><img src="" alt=""></a></li>
-            <li><a href=""></a><img src="" alt=""></li>
-            <li><a href=""><img src="" alt=""></a></li>
-            <li><a href=""><img src="" alt=""></a></li>
+        <img class="background" src="assets/img/background-header.png" alt="">
+        <img class="background-header-desktop" src="assets/img/back-eader-desktop.svg" alt="">
+        <div class="contain-nav-logo">
+            <img src="assets/img/logo-ypf.svg" alt="logo-ypf">
+            <nav class="nav-mobile">
+                <ul>
+                    <li><a href=""><img src="assets/icons/menu.svg" alt=""></a></li>
```

Comment:
Recordemos agregar los alts necesarios en las imágenes importantes con nombres que tengan semántica para describir qué imagen es la que no se puede visualizar por el navegador o para personas no videntes

```suggestion
                    <li><a href=""><img src="assets/icons/menu.svg" alt="logo-menu"></a></li>
```

Reply:
agregado